velox/functions/sparksql/tests/XxHash64Test.cpp

```diff
@@ -28,6 +28,7 @@ class XxHash64Test : public SparkFunctionBaseTest {
   }
 };
 
+// expected result get from spark-sql=> select xxhash64("Spark")
```

Comment:
Comments must be full sentences, i.e. start with a capital letter and end with a period. Please, update throughout.

```
The expected result was obtained by running SELECT xxhash64("Spark") query using spark-sql CLI.
```